cloudconfig/instancecfg/instancecfg.go

```diff
@@ -685,23 +655,38 @@ func NewInstanceConfig(
 		APIInfo:       apiInfo,
 		ImageStream:   imageStream,
 	}
-	return icfg, nil
 }
 
 // NewBootstrapInstanceConfig sets up a basic machine configuration for a
 // bootstrap node.  You'll still need to supply more information, but this
 // takes care of the fixed entries and the ones that are always needed.
 func NewBootstrapInstanceConfig(
+	confPath string,
+	tempPath string,
+	dataPath string,
+	logPath string,
+	metricsSpoolPath string,
 	config controller.Config,
 	cons, modelCons constraints.Value,
+	os os.OSType,
```

Comment:
Again, given that `NewInstanceConfig` doesn't use `os`, this doesn't need it either.